core/ecschema-editing/src/Editing/Mutable/MutableSchemaItem.ts

```diff
@@ -9,6 +9,7 @@ import {SchemaItem} from "@itwin/ecschema-metadata";
  * @internal
  */
 export abstract class MutableSchemaItem extends SchemaItem {
+  public abstract override setName(name: string): void;
```

Comment:
We should call this method setSchemaItemName to indicate that you'll set the name of a schema item with this method and not the name of a schema.